helmfile/values/falco.yaml.gotmpl

```diff
@@ -195,6 +195,18 @@ customRules:
         )
     - rule: Non sudo setuid
       enabled: false
+    - rule: The docker client is executed in a container
+      desc: Detect a k8s client tool executed inside a container
+      condition: spawned_process and container and not user_known_k8s_client_container_parens and proc.name in (k8s_client_binaries)
+      output: "Docker or kubernetes client executed in container (user=%user.name user_loginuid=%user.loginuid %container.info parent=%proc.pname cmdline=%proc.cmdline pid=%proc.pid image=%container.image.repository:%container.image.tag)"
+      priority: WARNING
+      tags: [container, mitre_execution]
+      exceptions:
+        - name: tekton_originated
+          fields: [k8s.ns.name]
+          comps: [=]
+          values:
+            - [tekton-pipelines]
```

Comment:
Please do append the exceptions in the `k8s_client_binaries` above use the same style as the rest have.